akka-bbb-apps/src/main/scala/org/bigbluebutton/core2/message/senders/MsgBuilder.scala

```diff
@@ -522,6 +522,60 @@ object MsgBuilder {
     BbbCommonEnvCoreMsg(envelope, event)
   }
 
+  def buildCamBroadcastStopSysMsg(

```

Comment:
L525-L561: those were recently removed from this file.
They might've been re-added by mistake on conflict resolution.